test/conformance/conformance_test.go

```diff
@@ -48,12 +48,12 @@ func TestGatewayAPIConformance(t *testing.T) {
 	}
 
 	cSuite := suite.New(suite.Options{
-		Client:                     client,
-		GatewayClassName:           *flags.GatewayClassName,
-		Debug:                      *flags.ShowDebug,
-		CleanupBaseResources:       *flags.CleanupBaseResources,
-		ValidUniqueListenerPorts:   validUniqueListenerPorts,
-		EnableAllSupportedFeatures: true,
+		Client:                   client,
+		GatewayClassName:         *flags.GatewayClassName,
+		Debug:                    *flags.ShowDebug,
+		CleanupBaseResources:     *flags.CleanupBaseResources,
+		ValidUniqueListenerPorts: validUniqueListenerPorts,
+		SupportedFeatures:        suite.StandardCoreFeatures, // ignore mesh conformance tests
```

Comment:
you want to add `Mesh` to `ExemptFeatures` instead, since core runs a small subset of tests and we want to run all